operators/operating-pinot/rebalance/rebalance-servers.md

```diff
@@ -155,15 +155,24 @@ Typically, the flags that need to be changed from defaults are
 | bestEfforts          | false         | <p>Applicable for rebalance with downtime=false.</p><p>If a no-downtime rebalance cannot be performed successfully, this flag <strong>controls whether to fail the rebalance or do a best-effort rebalance</strong>.</p>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 | reassignInstances    | false         | Applicable to tables where the instance assignment has been persisted to zookeeper. Setting this to true will make the rebalance **first update the instance assignment, and then rebalance the segments**.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
 | bootstrap            | false         | Rebalances all segments again, **as if adding segments to an empty table**. If this is false, then the rebalance will try to minimize segment movements.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
 
+### Rebalance with only 1 replica
+In general, when the table config uses only 1 replica, the downtime may be affected.
```

Comment:
i think with downtime = false and replication = 1 (with no changes to final replication count), the rebalance will itself be blocked. So this would be untrue "In this case, there'll be a downtime during rebalance"

Reply:
What do you mean with _the rebalance itself will be blocked_?

Reply:
I've recently found this PR is still open. I'm going to apply the change suggested by Manish. About the suggestion from Neha... do we know the actual behavior? IIRC I didn't actually tested that and the written here is what I inferred seen the code, but I may be wrong.

Reply:
@npawar, is this one good to merge re @gortiz comment above?